### ISSUE TITLE
Feature Request: Alternate click one monitor and disable all other monitors

### ISSUE THREAD

**Issue by zak123** [AlonX2/LightsOut#8]
Love this little utility, one ask I have is allowing an alternate click to disable my other two monitors. 

Appreciate your work!

**Comment by AlonX2** [comment]
Hey, glad your'e enjoying this utility!

Just to clarify, you'd like that an alt-click on a monitor would disable all monitors but that one?

**Comment by zak123** [comment]
Yeah! Or maybe even a configurable hot key that turns off all the monitors except the one my mouse is one, that would be even better to be honest.

**Comment by purduepete** [comment]
Great app - thank you!

Would love a global hot key, that toggles associated monitor on or off
